src/app/organizations/state/register-organization.service.ts

```diff
@@ -178,6 +189,7 @@ export class RegisterOrganizationService {
         location: organizationFormState.location,
         email: organizationFormState.contactEmail,
         status: Organization.StatusEnum.PENDING,
+        avatarUrl: this.returnImgLink(organizationFormState.avatarUrl),
```

Comment:
Try `avatarUrl: organizationFormState.avatarUrl || null` and get rid of function if it works

Reply:
Yes that works. Will remove.